prober/http.go

```diff
@@ -247,6 +333,42 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 	if resp == nil {
 		resp = &http.Response{}
 	}
+	i := 0
```

Comment:
You don't need a separate i now that tt.traces is a list, it's the first variable from the range.

Reply:
Good catch. Still a left over from earlier where this wasn't needed.